tests/integration/goldens/eventarc/google/cloud/eventarc_v1/types/channel.py

```diff
@@ -102,9 +102,9 @@ class State(proto.Enum):
                 possible cases this state can happen:
 
                 1. The SaaS provider disconnected from this
-                Channel. 2. The Channel activation token has
-                expired but the SaaS provider    wasn't
-                connected.
+                    Channel.
+                2. The Channel activation token has expired but
+                    the SaaS provider    wasn't connected.
```

Comment:
Weird whitespace here (was here before). Separate issue to be aware of.

Reply:
Filed https://github.com/googleapis/gapic-generator-python/issues/1754